src/config.js

```diff
@@ -11,9 +11,14 @@ datasets.forEach(d => {
   dateToDataset[d.date] = d
 })
 
+const theyWorkForUs = {
+  peoplePath: "https://theyworkforus.elect.in.th/people/"
```

Comment:
What do you think if we make this attribute a function?
For example,
```
peoplePath: (name) => name.trim().replace(" ", "-")
```

Then, we can use `config.theyWorkForUs.peoplePath(name)`.

Reply:
Updated in 2557bc6. I'm not fluent in JS so feel free to drop any comments!